src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -773,17 +771,25 @@ protected void HandleOutputDataReceived(string data)
         {
             try
             {
+                if (string.IsNullOrEmpty(data))
+                {
+                    // A null/empty data string indicates a problem in the transport,
+                    // e.g., named pipe emitting a null packet because it closed or some reason.
+                    // In this case we simply ignore the packet.
+                    return;
+                }
+
                 // Route protocol message based on whether it is a session or command message.
                 // Session messages have empty Guid values.
```

Comment:
Please remove this comment.
I mean please just remove this comment:
`// Session messages have empty Guid values.`
because it seems unnecessary now.

Reply:
Done.